ui/components/app/flask/experimental-area/index.scss

```
@@ -0,0 +1,44 @@
+.experimental-area {
+  color: $flask-purple;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  .logo {
+    padding: 16px;
+    line-height: 0.625rem;
+    font-family: monospace;
+  }
+
+  .experimental-text {
+    padding: 16px;
+    font-family: monospace;
+    margin: auto;
+    line-height: 16px;
+  }
+
+  .text {
+    padding: 16px;
+    max-width: 670px;
```

Includes a design system sass mixin for the body text
```suggestion
    max-width: 670px;
    @include H5;
```